tests/e2e/cypress/integration/Resources-status/resourceStatus.ts

```diff
@@ -0,0 +1,112 @@
+import { Given, And, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+import {
+  containerStateFilter,
+  selectFilterOnInput,
+  labelClearFilter,
+} from './resourceCommon';
+
+const serviceName = 'Ping';
+const resourcesRegexApiUrl = /\/api\/.+\/monitoring\/resources?/;
+
+Given('a valid centreon user account', () =>
+  cy.getCookie('PHPSESSID').should('exist'),
+);
+And('there is a page with a list of resources under monitoring', () =>
+  cy.get('table.MuiTable-root th p.MuiTypography-root').contains('Resource'),
```

Comment:
This is too brittle and doesn’t text anything rather than there is a table header named "Resource"